auto_install/install.sh

```diff
@@ -63,13 +64,21 @@ dhcpcdFile=/etc/dhcpcd.conf
 
 # Next see if we are on a tested and supported OS
 function noOS_Support() {
+    if [["${quietDistroCheck}" = true]]; then
+        echo "::: Exiting due to an invalid OS"
+        exit 1
+    fi
     whiptail --msgbox --backtitle "INVALID OS DETECTED" --title "Invalid OS" "We have not been able to detect a supported OS.
 Currently this installer supports Raspbian and Debian (Jessie and Stretch), Devuan (Jessie) and Ubuntu from 14.04 (trusty) to 17.04 (zesty).
 If you think you received this message in error, you can post an issue on the GitHub at https://github.com/pivpn/pivpn/issues." ${r} ${c}
     exit 1
 }
 
 function maybeOS_Support() {
+    if [["${quietDistroCheck}" = true]]; then
+        echo "::: Exiting due to unsupported OS"
```

Comment:
Here I think we should better clarify users that the OS Is possibly supported, 
however, for the sake of unattended install,  the script is bailing out because we cannot be sure of it.